### PR TITLE
Add Dev Preview updates in Changelog

### PR DESCRIPTION
**Pull request by tmattio**
And add buttons in the Changelog header to select important tags.

Suggested by @shonfeder.

@sabine I've added a tag `ocaml-org`. Are there any updates we want to incorporate from Discuss as part of this PR, or should I remove it for now?

### CODE REVIEW

File: data/changelog/dune/2024-09-25-call-for-feedback.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+---
+
+Hello folks! 👋 
+
+We'd like to welcome everyone to try and play with the [Dune Developer Preview](https://dune.ci.dev)! 🎉 
+
+This experimental nightly release of dune includes a lot of improvements, including the much expected package management features, and it can be installed from that website or by using the new installation script:
+
+```bash
+$ curl https://dune.ci.dev/install | bash
```
Comment (Leonidas-from-XIV) [inline data/changelog/dune/2024-09-25-call-for-feedback.md]:
```suggestion
$ curl -fsSL https://get.dune.build/install | bash
```

File: data/changelog/dune/2024-09-25-call-for-feedback.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
+---
+title: Call for Feedback
+tags: [dune, platform, developer-preview]
+---
+
+Hello folks! 👋 
+
+We'd like to welcome everyone to try and play with the [Dune Developer Preview](https://dune.ci.dev)! 🎉
```
Comment (Leonidas-from-XIV) [inline data/changelog/dune/2024-09-25-call-for-feedback.md]:
```suggestion
We'd like to welcome everyone to try and play with the [Dune Developer Preview](https://preview.dune.build)! 🎉

File: data/changelog/dune/2024-09-25-call-for-feedback.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+#### FunOCaml Presentation
+At **FunOCaml** we had a last-minute opportunity to present the work being done on Dune and we used it to introduce the Developer Preview to the community, and even tested Package Management live with suggestions from the audience (thanks @anmonteiro and Paul-Elliot for participating!) – you can [watch it on Twitch](https://www.twitch.tv/videos/2252408016?t=08h12m00s).
+
+#### New design
+We're working with @Claire_Vandenberghe on redesigning the Developer Preview website so that it'd feel like a seamless extension of OCaml.org – in this current iteration we've made it easier to get started and we're putting the FAQ front and center.
+
+We'll be iterating on this design in the coming weeks until it fits perfectly within the OCaml.org design system 🎨 
+
+You can check the new website here: https://dune.ci.dev
```
Comment (Leonidas-from-XIV) [inline data/changelog/dune/2024-09-25-call-for-feedback.md]:
```suggestion
You can check the new website here: https://preview.dune.build
```

**Review by Leonidas-from-XIV — CHANGES_REQUESTED** [review]
I think we should use the new URLs in the post. Especially the `curl` part will not work, despite us setting up a redirect as `curl` without `-L` does not follow redirects.

**Comment by sabine** [comment]
@tmattio What kind of updates from Discuss are you considering?

**Comment by tmattio** [comment]
We could add the launch announcement https://discuss.ocaml.org/t/v3-ocaml-org-we-are-live/9747, for instance, and any other post that you think would be relevant. It could be highlights extracted from the newsletters if we want content.

**Comment by sabine** [comment]
Only need to be aware that anything published within the last 90 days will go out through the social media accounts via the RSS feed. As long as this is not a problem... historic announcements could be added.

**Review by christinerose — APPROVED** [review]

**Pull request merged**